release/sdk-config.yaml.tmpl

```
@@ -47,11 +47,11 @@ commands:
 - name: sandbox
   path: daml-helper/daml-helper
   desc: "Launch Sandbox"
-  args: ["run-platform-jar", "--logback-config=sandbox-logback.xml", "sandbox"]
+  args: ["run-jar", "--logback-config=sandbox-logback.xml", "sandbox"]
 - name: sandbox-classic
   path: daml-helper/daml-helper
   desc: "Deprecated. Launch Sandbox Classic (the default Sandbox implementation for SDK <= 0.13.55)"
-  args: ["run-platform-jar", "--logback-config=sandbox-logback.xml", "sandbox-classic"]
+  args: ["run-jar", "--logback-config=sandbox-logback.xml", "sandbox-classic"]
```

Oh right, these need to go like,

```suggestion
  args: ["run-jar", "--logback-config=sandbox-logback.xml", "daml-sdk/daml-sdk.jar", "sandbox-classic"]
```

---

duh ofc they do sorry I’ll test this time before pushing :facepalm: